src/main/kotlin/org/jetbrains/research/testspark/tools/ToolUtils.kt

```diff
@@ -138,10 +138,14 @@ object ToolUtils {
      *
      * @return true if the process has been stopped, false otherwise
      */
-    fun isProcessStopped(project: Project, indicator: CustomProgressIndicator): Boolean {
-        if (project.service<ErrorService>().isErrorOccurred()) return true
+    fun isProcessStopped(errorMonitor: ErrorMonitor, indicator: CustomProgressIndicator): Boolean {
+        if (errorMonitor.isErrorOccurred) return true
```

Comment:
I think it is better to either make it a method or make the variable's setter private.

Reply:
Fixed in 789420a66103d53c7e826a9de1847220e517ca10

Reply:
'+